deluge/core/preferencesmanager.py

```diff
@@ -210,8 +210,7 @@ def __set_listen_on(self):
             self.config['listen_random_port'] = None
             listen_ports = self.config['listen_ports']
 
-        interface = str(self.config['listen_interface'].strip())
-        interface = interface if interface else '0.0.0.0'
+        interface = str(self.config['listen_interface'].strip()) if self.config['listen_interface'] else '0.0.0.0'
```

Comment:
The str() call can be removed and I would prefer a multi-line if statement here.

Reply:
Done